ports/espressif/boards/m5stack_stamps3/board.h

```diff
@@ -0,0 +1,116 @@
+ * THE SOFTWARE.
+ */
+
+// #ifndef ADAFRUIT_FEATHER_ESP32S3_TFT_H_
+// #define ADAFRUIT_FEATHER_ESP32S3_TFT_H_
+// #ifndef ADAFRUIT_FEATHER_ESP32S3_NOPSRAM_H_
+// #define ADAFRUIT_FEATHER_ESP32S3_NOPSRAM_H_
+
+// #define CFG_UF2_FLASH_SIZE          (8*1024*1024)
+// #define BOARD_FLASH_SIZE      (8*1024*1024)
+
+
+//--------------------------------------------------------------------+
+// Button
+//--------------------------------------------------------------------+
+
+// Enter UF2 mode if GPIO is pressed while 2nd stage bootloader indicator
+// is on e.g RGB = Purple. If it is GPIO0, user should not hold this while
+// reset since that will instead run the 1st stage ROM bootloader
+#define PIN_BUTTON_UF2        0
+
+// GPIO that implement 1-bit memory with RC components which hold the
+// pin value long enough for double reset detection.
+#define PIN_DOUBLE_RESET_RC   38
+
+//--------------------------------------------------------------------+
+// LED
+//--------------------------------------------------------------------+
+
+// GPIO connected to Neopixel data
+#define NEOPIXEL_PIN          21
+
+// #define NEOPIXEL_POWER_PIN    34
+// #define NEOPIXEL_POWER_STATE  1
+
+// Brightness percentage from 1 to 255
+#define NEOPIXEL_BRIGHTNESS   0x10
+
+// Number of neopixels
+#define NEOPIXEL_NUMBER       1
+
+// LED for indicator and writing flash
+// If not defined neopixel will be use for flash writing instead
+// #define LED_PIN               13
+// #define LED_STATE_ON          1
+
+//--------------------------------------------------------------------+
+// TFT
+//--------------------------------------------------------------------+
+
+#define CONFIG_LCD_TYPE_ST7789V
+
+#define DISPLAY_PIN_MISO      -1 // required if use CONFIG_LCD_TYPE_AUTO
+#define DISPLAY_PIN_MOSI      35
+#define DISPLAY_PIN_SCK       36
+
+#define DISPLAY_PIN_CS        37
+#define DISPLAY_PIN_DC        34
+#define DISPLAY_PIN_RST       33
+
+#define DISPLAY_PIN_BL        45
+#define DISPLAY_BL_ON          1  // GPIO state to enable back light
+
+#define DISPLAY_PIN_POWER     21
+#define DISPLAY_POWER_ON       1  // GPIO state to enable TFT
+
+#define DISPLAY_WIDTH         240
+#define DISPLAY_HEIGHT        135
+
+#define DISPLAY_COL_OFFSET    53
+#define DISPLAY_ROW_OFFSET    40
+
+// Memory Data Access Control & // Vertical Scroll Start Address
+#define DISPLAY_MADCTL        (TFT_MADCTL_MX)
+#define DISPLAY_VSCSAD        0
+
+#define DISPLAY_TITLE         "M5 Stamp S3 TFT"
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID                  0x303A
+#define USB_PID                  0x811A
+
+#define USB_MANUFACTURER         "M5"
+#define USB_PRODUCT              "Stamp S3"
+
+#define UF2_PRODUCT_NAME         USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID             "ESP32S3"
```

Comment:
incorrect board-id format

Reply:
I have updated the board folder from cardputer to stamps3 to better match the board.

Reply:
board-id still not in correct format

Reply:
```#define UF2_BOARD_ID             "ESP32S3-StampS3-01"```

Here is the correct board id, i didnt want to make another PR